cmd/go.mod

```diff
@@ -7,7 +7,7 @@ toolchain go1.24.1
 require (
 	github.com/defenseunicorns/go-oscal v0.6.2
 	github.com/goccy/go-yaml v1.17.1
-	github.com/revanite-io/sci v0.3.5-0.20250506145353-e4f84ef5e40b
+	github.com/revanite-io/sci v0.3.6
```

Comment:
This and the go.sum change are in #307, which is awaiting a second maintainer approval. Practically, it doesn't matter which PR wins the race so I'm not asking you to change this one, but in general let's keep PRs to one logical change.